.gitignore

```diff
@@ -7,7 +7,7 @@
 TAGS
 config.json
 node_modules
-package-lock.json
+
```

Comment:
We don't need to check in package-lock.json files -- things are changing rapidly still.

Reply:
package dependencies should not be rapidly changing, and if they are, package-lock should be committed to avoid ambiguity... https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5